tap_salesforce/salesforce/bulk.py

```diff
@@ -107,7 +107,7 @@ def _can_pk_chunk_job(self, failure_message): # pylint: disable=no-self-use
 
     def _bulk_query(self, catalog_entry, state):
         job_id = self._create_job(catalog_entry)
-        start_date = self.sf.get_start_date(state, catalog_entry)
+        start_date = self.sf.get_start_date(state, catalog_entry, with_lookback=True)
```

Comment:
Remove `with_lookback` because it should not exist as a parameter in the new implementation

Reply:
Removed `with_lookback` param from all places.